src/main/java/nom/tam/fits/compression/algorithm/rice/RiceQuantizeCompressOption.java

```diff
@@ -39,6 +40,10 @@ public RiceQuantizeCompressOption() {
         super(new RiceCompressOption());
     }
 
+    public RiceQuantizeCompressOption(ICompressOption compressOption) {
```

Comment:
This may be fine, but if there is a way to restrict the argument to `RiceCompressOption` class it would be better, as it would prevent some misuse. Of cousre, if it's invoked from a generic code where the options class is not given, it may not help...

Reply:
Oh, and while at it, you probably want to do the equivalent for `HCompressQuantizeOption` also (regardless of which method signature you settle on.)